client/ayon_blender/api/plugin.py

```diff
@@ -295,7 +299,7 @@ def collect_instances(self):
 
         # Process only instances that were created by this creator
         for instance_node in cached_instances.get(self.identifier, []):
-            property = instance_node.get(AVALON_PROPERTY)
+            property = instance_node.get(AYON_PROPERTY)
```

Comment:
This is guess, shouldn't this also have the fallback?
```suggestion
            property = instance_node.get(AYON_PROPERTY)
            if not propery:
                property = instance_node.get(AVALON_PROPERTY)
```